torch/jit/__init__.py

```diff
@@ -663,11 +664,10 @@ def trace(func,
                                      var_lookup_fn, _force_outplace)
 
     # Check the trace against new traces created from user-specified inputs
-    if check_trace:
-        if check_inputs is not None:
-            _check_trace(check_inputs, func, executor_options, module, check_tolerance, _force_outplace)
-        else:
-            _check_trace([example_inputs], func, executor_options, module, check_tolerance, _force_outplace)
```

Comment:
@jamesr66a And the original code made zero sense. This line will never be executed.

Reply:
@SsnL these are two different names, `check_inputs` and `check_trace`